api/common/env.go

```diff
@@ -0,0 +1,74 @@
+package common
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// GetEnv looks up a key under its name in env or name+_FILE to read the value
+// from a file. fallback will be defaulted to if a value is not found.
+func GetEnv(key, fallback string) string {
```

Comment:
we've declared these in a few places now, this is from fiddling, but gonna try to drive this wedge in here ;)

I'll go through in another PR and replace all usage with these.